src/lua/LuaInterfaceGame.cpp

```diff
@@ -140,6 +140,9 @@ LuaInterfaceGame::LuaInterfaceGame(GameWorldGame& gw) : gw(gw)
     lua["RES_GRANITE"] = Resource::Granite;
     lua["RES_WATER"] = Resource::Water;
 
+    lua["PACTTYPE_NON_AGGRESSION"] = PactType::NON_AGGRESSION_PACT;
```

Comment:
What about simply `NON_AGGRESSION_PACT`?

Reply:
I have changed both, so ALLIANCE is also TREATY_OF_ALLIANCE now.